nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -174,25 +175,28 @@ def _impl(inputs, attr, params):
         input_shapes = attr['_input_shapes'][inputs[0]]
 
         # Extract kernel shape from params
-        conv_param_weights = params[inputs[1].list_output_names()[0]]
+        if inputs[1] in attr['_input_shapes']:
```

Comment:
Can you share details of use case where convolution weights not in params ?

Reply:
This case is observe while running [AlexNet](https://www.cs.toronto.edu/~guerzhoy/tf_alexnet/) model.

Reply:
Is weights output of another node in runtime?

Can u share serialised tensor flow graph?

Reply:
fine tuned [Alexnet](https://kratzert.github.io/2017/02/24/finetuning-alexnet-with-tensorflow.html) can be found here for the above reference